add simons refined samples

Veluchs opened the pull request:
does not include the samples which rely on external test data.

Veluchs committed:
add simons refined samples

does not include the samples which rely on external test data

terryyz commented:
Hi Simon!

Could you please also add your split to `data/raw' and run the pytest workflows in your forked repo?

To set up the workflows, you need to enable GitHub Actions in your repo and may replace the `script/run.sh` with the following content:

```sh
NAMES=(chien jenny wenhao niklas hanhu ratna simon)
for name in "${NAMES[@]}"; do
    cp data/raw/*"$name"*py data/clean
done
flake8 data/clean/*.py --select=E9,F63,F7,F82 --show-source --statistics
python script/parse.py

for name in "${NAMES[@]}"; do
    for file in data/processed/*"$name"*wo_doc.py; do

        if ! pytest "$file"; then
            echo "Pytest failed on $file, stopping..."
            exit 1
        fi
    done

    for file in data/processed/*"$name"*w_doc.py; do
        if ! pytest --doctest-modules "$file"; then
            echo "Pytest failed on $file, stopping..."
            exit 1
        fi
    done
done
```

Cheers

terryyz commented on data/clean/f_648_simon.py:
```diff
@@ -0,0 +1,115 @@
+    - sm.regression.linear_model.RegressionResultsWrapper: The result of the OLS regression, or None if no rows meet the criteria or DataFrame is empty.
+
+    Requirements:
+    - pandas for DataFrame manipulation.
```
please remove any NL descriptions for the required modules like this one

Veluchs replied:
done!

Veluchs committed:
add simon

Veluchs commented:
Should I add my finalized samples to raw or the initial ones I received?

Veluchs committed:
add requirements

Veluchs committed:
remove library description

Veluchs committed:
add requirements

terryyz commented:
> Should I add my finalized samples to raw or the initial ones I received?

Finalized ones :)

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
add requirements

Veluchs committed:
remove module description

Veluchs committed:
add requirements

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
remove module description

Veluchs committed:
fix annotation

Veluchs committed:
add updated to raw

Veluchs committed:
remove external test data

Veluchs commented:
![image](https://github.com/bigcode-project/open-eval/assets/135350576/621f3cd7-f433-4660-b169-fb0cafd9d638)

I'm running into this error. Seems to me that there is a file missing. Can you help me out?

terryyz commented:
> ![image](https://private-user-images.githubusercontent.com/135350576/325727346-621f3cd7-f433-4660-b169-fb0cafd9d638.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwNjk3MjIsIm5iZiI6MTcxNDA2OTQyMiwicGF0aCI6Ii8xMzUzNTA1NzYvMzI1NzI3MzQ2LTYyMWYzY2Q3LWY0MzMtNDY2MC1iMTY5LWZiMGNhZmQ5ZDYzOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNVQxODIzNDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNmQ0OWZkNGU0ZTEwMDkwZjgwNGNlY2IwNWQyZGYyZDgxMmMyYWQ2YzM1NzMzYTJmOTA5MTgzZTU4NjI4YThmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.X1378CyROxR1Qzop20P2TjjrGegMmj1sb1M0ETrPwf8)
> 
> I'm running into this error. Seems to me that there is a file missing. Can you help me out?

Just fixed `script/parse.py`. Please git pull and run the actions again :)

Veluchs committed:
Merge branch 'bigcode-project:main' into main

Veluchs committed:
remove empty test

Veluchs committed:
Merge branch 'main' of https://github.com/Veluchs/open-eval

Veluchs commented:
![image](https://github.com/bigcode-project/open-eval/assets/135350576/9589b983-cdbb-45be-b20d-1ea59ca03fac)

I'm running into this error. The function creates a log file using the logging module. Locally the test run just fine. Any idea why that happens?

terryyz commented:
> ![image](https://private-user-images.githubusercontent.com/135350576/325740898-9589b983-cdbb-45be-b20d-1ea59ca03fac.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwNzM2NzksIm5iZiI6MTcxNDA3MzM3OSwicGF0aCI6Ii8xMzUzNTA1NzYvMzI1NzQwODk4LTk1ODliOTgzLWNkYmItNDViZS1iMjBkLTFlYTU5Y2EwM2ZhYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQyNVQxOTI5MzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMTZiNmY5ZDQxYjdmZDJkNDFhM2E5MmM2MmY5ZmRkYjdmNTNmY2E0MmU5ZTk1ZTNkMTVlM2YzOTUxMmUyOTM4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.fM8b5DIFDQHXdiMSoG34fLvKv3QviJkEWl4HkXi2cbw)
> 
> I'm running into this error. The function creates a log file using the logging module. Locally the test run just fine. Any idea why that happens? I could probably add a save path for the log file though, I guess that would work.

I don't the specific reasons, unfortunately. However, I'd suggest passing a specific path for the function to write the log file. You can set up the dummy directory in the test cases and tear it down later :)

Veluchs committed:
fix requirements

Veluchs committed:
fix return

Veluchs committed:
add examples

Veluchs committed:
add temp directories for testing

Veluchs committed:
refine examples

Veluchs committed:
fix examples

Veluchs committed:
fix examples

Veluchs committed:
fix example output

Veluchs committed:
fix examples

Veluchs committed:
fix example output

Veluchs committed:
fix test

Veluchs committed:
remove samples from clean

Veluchs committed:
fix temporary directory creation

Veluchs committed:
fix examples

Veluchs committed:
fix examples

Veluchs committed:
fix example output

Veluchs committed:
fix example output

Veluchs committed:
fix examples

Veluchs committed:
fix example

Veluchs committed:
fix dtype

Veluchs committed:
add sympy

Veluchs committed:
fix typo

Veluchs committed:
fix examples

Veluchs committed:
Merge branch 'main' of https://github.com/Veluchs/open-eval

Veluchs committed:
fix

Veluchs committed:
fix output

Veluchs committed:
fix

Veluchs committed:
fix

Veluchs committed:
fix

Veluchs committed:
f

Veluchs committed:
add seed

Veluchs committed:
f

Veluchs committed:
f

Veluchs committed:
add additional libary usage

Veluchs committed:
add explicit second library usage

Veluchs committed:
fix test case

Veluchs commented:
Hey,

all test conclude successfully now. Should be ready to merge.

terryyz commented:
Hi @Veluchs, could you please resolve your `data/processed` by updating your repo to the latest version? I also found that you pushed quite a few external data to the repo. Could you also remove them? Thanks!

Veluchs committed:
remove external data

Veluchs committed:
Merge branch 'main' of https://github.com/bigcode-project/open-eval

Veluchs committed:
remove unused libraries

Veluchs commented:
> Hi @Veluchs, could you please resolve your `data/processed` by updating your repo to the latest version? I also found that you pushed quite a few external data to the repo. Could you also remove them? Thanks!

Hi @terryyz, 
now it should be ready to merge. Removed the external data which I pushed accidentally and also updated to the latest version.

terryyz committed:
Merge branch 'main' into main

terryyz committed:
remove: remove directorio/custom1.txt

terryyz committed:
remove: remove directorio/custom2.txt
